### PR TITLE
First step in introducing other libraries

### PR DESCRIPTION
**Pull request by valcikeric**
I added nav to change the showed library, but I have not added any errors to the other libraries. So currently the three new libraries are marked as not yet supported and they can't be clicked or accessed.

I also changed the structure. My vision right now is that each library is going to have its own page, but those pages will contain everything the current index main page contains, only the errors will change. The structure is somewhat changed so this will be easier to implement. Please, feel free to share your ideas or comment on the things you don't like.

I also came across another idea. We could make the errors change on the same page using JavaScript as showed [here](https://getbootstrap.com/docs/4.0/components/navs/#javascript-behavior). Let me know if you'd like this.

These changes are mergeable IMO, so this PR is ready to be reviewed.

### CODE REVIEW

**Comment by valcikeric** [comment]
I just pushed another commit. I looked into the JavaScript navigation and content switching and I find a lot better than the first idea. Also, this solution adds very little to the current structure compared to the first one.

If you want to see yourself how it looks and feels, just delete the `disabled: true` lines in `_data/libraries.yml` and build the website. The commit I pushed has all the new libraries disabled because they are not finished yet.

**Comment by valcikeric** [comment]
Alright, that's no problem, I see this as my mistake. I shouldn't have gone on my own and do so much work without consulting you.

I didn't know about the conservative use of JS, and that the website should work even without the use of JS. I see that that's a problem in the JavaScript solution I proposed.

About the graphics, until now, I haven't even noticed the subtle background-color changes. Both implementations I created break the current background-color layout. Next time I will take care of that as well.

Anyways, I see your point. I will work on the changes you mentioned in the last but one paragraph and if I understood it correctly, I won't touch the page's design or layout. I don't really have any questions about this, but I'm not really sure about the next steps after that, so I think that a call would be very beneficial. I realized that if I would somehow add things to the current design and layout of the page, I would need some guidance.

So TL;DR, no call is needed right now, and I will work on the things you mentioned.
One final question, should I push the changes to this PR or create a new one?

**Comment by mukrop** [comment]
OK, let's have a call in some time. Late next week or early the week after that?

As for branching – make a separate branch and corresponding PR, but please keep this one in, I'll be thinking about the layout and this is a useful draft for the tests,

**Comment by valcikeric** [comment]
I will most likely be away from Wednesday for a week. And if you don't mind, I'd prefer to have the call earlier than later. So at the start of next week, maybe Tuesday?

**Comment by mukrop** [comment]
Agreed that sooner is better than later, I've sent you an invite for Tuesday 10:00 call (let me know if the time does not suit you).

**Comment by valcikeric** [comment]
That's fine with me.

**Comment by mukrop** [comment]
I'm closing this one as the changes will be done differently (as discussed on the call). The code will still exist in the repo if needed.
